package-lock.json

```diff
@@ -55703,6 +55704,7 @@
       "version": "file:packages/compass-connections-navigation",
       "requires": {
         "@mongodb-js/compass-components": "^1.25.1",
+        "@mongodb-js/compass-connections": "^1.30.0",
```

Comment:
how did this happen?

Reply:
its probably because I did a bootstrap and the lock file was not in sync?